examples/move/transfer-to-object/README.md

```diff
@@ -0,0 +1,219 @@
+`Register` object and delays in payment processing. In contrast, with
+Implementation 1, since it uses only owned objects, all payments across all of
+your restaurant locations could be processed in parallel and not effect each
+other.
+
+Luckily, transfer-to-object can help parallelize the payment process to the
+`Register` object while also keeping the benefits of dynamic authorization and
+stable interaction IDs that we saw in this implementation. Let's examine
+exactly how it does this in the next example.
+
+You can see the Move implementation for this section [here](./shared-no-tto/sources/shared_cash_register.move).
+
+## Implementation 3: Using a Shared Object + Transfer-to-Object
+
+With transfer-to-object, we can combine the benefits of the two previous implementations:
+
+- The object ID stability of the shared object register.
+- The ability to transfer the ownership of the object in case of key compromise.
+- An easy way of dynamically adding, removing, and enforcing permissions on who
+  can withdraw payments.
+- Payments can still be made using the `identified_payment::make_payment`
+  function that uses `sui::transfer::transfer` under the hood, so payments can
+  happen in parallel across all restaurant locations without needing to be
+  sequenced against the shared `Register` object.
+
+You can see the entire implementation for the shared object register using
+transfer-to-object [here](./shared-with-tto/sources/shared_cash_register.move).
+
+Let's go through this implementation in more detail and compare it to the above
+two implementations.
+
+### Interaction Stability: Object ID Remains the Same
+
+To make a payment, nothing changes from Implementation 1. In particular,
+customers will still use `identified_payment::make_payment` and simply set the
+address they want to send to be the object ID of the restaurant's `Register`
+object. If the restaurant changes the ownership of the `Register` object, this
+will be opaque to the customers – they will always send their payment to the
+same `Register` object.
+
+### Receiving Payments
+
+At a high level, handling payments after they have been made using
+transfer-to-object resides somewhere between both Implementation 1 and
+Implementation 2. In particular:
+
+- Similar to Implementation 1, the object IDs of the payments you want to
+  handle in that transaction will show up in the transaction's inputs.
+- Similar to Implementation 2, there are dynamic checks enforced on being able
+  to access the sent payments.
+
+To understand what's going on here, it's best to go through the implementation
+of `handle_payment`:
+
+```move
+/// We take the `Register` shared object mutably, along with a "ticket"
+// `handle_payment` that we can exchange for the actual `IdentifiedPayment` object
+// that it is associated with.
+public fun handle_payment(register: &mut Register, handle_payment: Receiving<IdentifiedPayment>, ctx: &TxContext): IdentifiedPayment {
```

Comment:
- Thoughts on just `receive_payment` here, to avoid a synonymous term here.
- We can't use `public_receive`, right, because `IdentifiedPayment` doesn't have `store`? (At least I hope so, because otherwise you can bypass the auth check).

Reply:
I had updated `IdentifiedPayment` to have `store` but forgot to update the definition here, so great catch (but this should work now)